backend/requirements.txt

```diff
@@ -1,3 +1,84 @@
+absl-py==0.7.1
+aniso8601==8.0.0
+asgiref==3.2.3
+astor==0.8.0
+astroid==2.1.0
+autopep8==1.4.3
+beautifulsoup4==4.8.0
+bokeh==1.4.0
+certifi==2019.6.16
+chardet==3.0.4
+Click==7.0
+colorama==0.4.1
+cycler==0.10.0
+Django==3.0.2
+docx==0.2.4
 Flask==1.1.1
+flask-mongoengine==0.9.5
+Flask-RESTful==0.3.7
+Flask-WTF==0.14.2
+gast==0.2.2
+google-pasta==0.1.7
+grpcio==1.22.0
+h5py==2.9.0
+idna==2.8
+imageio==2.5.0
+isort==4.3.4
+itsdangerous==1.1.0
+Jinja2==2.10.3
+Keras-Applications==1.0.8
+Keras-Preprocessing==1.1.0
+keyboard==0.13.3
+kiwisolver==1.1.0
+lazy-object-proxy==1.3.1
+lxml==4.4.0
+Markdown==3.1.1
+MarkupSafe==1.1.1
+matplotlib==3.1.1
+mccabe==0.6.1
+mongoengine==0.19.1
+mysql-connector-python==8.0.17
+mysqlclient==1.4.6
+numpy==1.16.4
+packaging==20.0
+pandas==0.25.3
+Pillow==6.1.0
+protobuf==3.8.0
+PyAutoGUI==0.9.45
+pycodestyle==2.5.0
+pygame==1.9.6
+PyGetWindow==0.0.6
+pylint==2.2.2
+pymongo==3.10.1
+PyMsgBox==1.0.7
+pynput==1.4.2
+pyparsing==2.4.2
+pyperclip==1.7.0
+PyRect==0.1.4
+PyScreeze==0.1.22
+python-dateutil==2.8.0
+python-docx==0.8.10
+PyTweening==1.0.3
+pytz==2019.3
+PyYAML==5.3
+random2==1.0.1
+requests==2.22.0
+robotframework==3.1.2
+scipy==1.3.1
+selenium==3.141.0
+signalrcore==0.7.9
+six==1.12.0
+soupsieve==1.9.2
+sqlparse==0.3.0
+tensorboard==1.14.0
+tensorflow==1.14.0
+tensorflow-estimator==1.14.0
```

Comment:
I believe that using tensorflow for our project is currently out of the scope

Reply:
Sorry it was added automatically, I removed it in last commit